swarm/testutil/http.go

```diff
@@ -97,12 +99,25 @@ func NewTestSwarmServer(t *testing.T, serverFunc func(*api.API) TestServer) *Tes
 
 type TestSwarmServer struct {
 	*httptest.Server
-	Hasher    storage.SwarmHash
-	FileStore *storage.FileStore
-	dir       string
-	cleanup   func()
+	Hasher            storage.SwarmHash
+	FileStore         *storage.FileStore
+	dir               string
+	cleanup           func()
+	timestampProvider *fakeTimeProvider
 }
 
 func (t *TestSwarmServer) Close() {
 	t.cleanup()
 }
+
+func (t *TestSwarmServer) GetCurrentTime() mru.Timestamp {
+	return t.timestampProvider.Now()
+}
+
+// EnableMetrics is starting InfluxDB reporter so that we collect stats when running tests locally
```

Comment:
@jpeletier isn't this creating a circular dependency? Are you actually using `EnableMetrics` anywhere?

Reply:
?? I didn't touch anything regarding that EnableMetrics. Only the code above `GetCurrentTime()` belongs to this change. Maybe @nolash knows.

Reply:
OK, I will remove it.